src/java/org/apache/cassandra/index/sai/disk/v1/IndexSearcher.java

```diff
@@ -39,13 +39,13 @@
  */
 public abstract class IndexSearcher implements Closeable
 {
-    final PrimaryKeyMap.Factory primaryKeyMapFactory;
-    final PerIndexFiles indexFiles;
-    final SegmentMetadata metadata;
-    final IndexDescriptor indexDescriptor;
-    final IndexContext indexContext;
+    protected final PrimaryKeyMap.Factory primaryKeyMapFactory;
+    protected final PerIndexFiles indexFiles;
+    protected final SegmentMetadata metadata;
+    protected final IndexDescriptor indexDescriptor;
+    protected final IndexContext indexContext;
 
-    IndexSearcher(PrimaryKeyMap.Factory primaryKeyMapFactory,
+    public IndexSearcher(PrimaryKeyMap.Factory primaryKeyMapFactory,
```

Comment:
nit: It probably makes sense to promote `IndexSearcher` to `org.apache.cassandra.index.sai.disk` since it isn't now strictly a `v1` construct.